### ISSUE TITLE
Parallel update Check to reduce completion time

### ISSUE THREAD

**Issue by Evan-aja** [tachiyomiorg/tachiyomi#7607]
### Describe your suggested feature

Tachiyomi should have the ability to do parallel chapter update.

Currently, I have to wait for around 3 minutes to refresh all of my galleries (220+) since the current implementation still does the update in serial.

This will help people ho have large collection of Manga whenever they decided to do a bulk update.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.13.5](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by TheUnlocked** [comment]
Manga within a source are intentionally not updated in parallel because it increases strain on the sources and can cause you to get rate limited/IP banned. Tachiyomi does update manga from different sources in parallel.

**Comment by alessandrojean** [comment]
Closing as it will not be implemented by the reasons explained above. Diversify your library and it will make more usage of the parallel update when having different sources.
